netconf_client/ncclient.py

```diff
@@ -21,40 +24,258 @@
 )
 
 
+# Defines the scope for netconf traces
+logger = logging.getLogger("netconf_client_pretty")
```

Comment:
Common convention I've see is for the logs to be scoped to the same name as the module.

```suggestion
logger = logging.getLogger("netconf_client.manager")
```